src/Conversions.cc

```diff
@@ -1390,6 +1390,12 @@ msgs::ParticleEmitter ignition::gazebo::convert(const sdf::ParticleEmitter &_in)
     header->add_value(_in.Topic());
   }
 
+  // todo(anyone) add particle_scatter_ratio field in particle_emitter.proto
```

Comment:
How about making a PR in ign-msgs targeting Fortress? Then this `todo` could be changed to:

```
Use particle_scatter_ratio in particle_emitter.proto from Fortress on.
```

Reply:
the field was added in https://github.com/ignitionrobotics/ign-msgs/pull/162 so updated comment 
99b8ae7